src/shared/dissect-image.c

```diff
@@ -1409,56 +1414,101 @@ int decrypted_image_relinquish(DecryptedImage *d) {
         return 0;
 }
 
-int root_hash_load(const char *image, void **ret, size_t *ret_size) {
+int verity_metadata_load(const char *image, void **roothash, size_t *roothash_size, char **hashdevice) {
```

Comment:
we generally prefix our return parameters with `ret_` to make clear that they are about output, not input. Hence please rename these `ret_root_hash`, `ret_root_hash_size` and `ret_verity_image` or so

Reply:
Ok, done